### PR TITLE
chore: all modules use new config

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

**Comment by tokers** [comment]
Test case for `kube. BuildRestConfig` will be added after we use terratest.

File: pkg/ingress/controller/apisix_service.go [hunk 1]
```diff
@@ -16,8 +16,8 @@ package controller
 
 import (
 	"fmt"
-	"github.com/api7/ingress-controller/pkg/ingress/apisix"
-	"github.com/golang/glog"
+	"time"
```
Comment (gxthrj) [inline pkg/ingress/controller/apisix_service.go:19]: The effect of this `time` was not found in this change.

File: pkg/ingress/controller/apisix_upstream.go [hunk 2]
```diff
@@ -16,9 +16,8 @@ package controller
 
 import (
 	"fmt"
-	"github.com/api7/ingress-controller/pkg/ingress/apisix"
-	"github.com/api7/ingress-controller/pkg/ingress/endpoint"
-	"github.com/golang/glog"
+	"time"
```
Comment (gxthrj) [inline pkg/ingress/controller/apisix_upstream.go:19]: ditto

File: pkg/kube/init.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	}
+
+	apisixKubeClient, err = clientSet.NewForConfig(restConfig)
+	if err != nil {
+		return err
+	}
+	CoreSharedInformerFactory = informers.NewSharedInformerFactory(kubeClient, 0)
+
+	return nil
+}
```
Comment (gxthrj) [inline pkg/kube/init.go:61]: It is a big change, we need to test if informers can work as before.

File: pkg/ingress/controller/apisix_service.go (hunk 1 above)
Comment (tokers) [reply pkg/ingress/controller/apisix_service.go:19]: image

It's used here.

File: pkg/ingress/controller/apisix_upstream.go (hunk 2 above)
Comment (tokers) [reply pkg/ingress/controller/apisix_upstream.go:19]: It's used here
image

File: pkg/kube/init.go (hunk 3 above)
Comment (tokers) [reply pkg/kube/init.go:61]: Well, it's not easy to write unit test cases now, i will run them in my local minikube.

Comment (gxthrj) [reply pkg/kube/init.go:61]: ok

**Review by gxthrj — APPROVED** [review]
LGTM

**Comment by tokers** [comment]
@gxthrj After this PR is merged, the dev docker image should be updated, also, docs and samples should be changed, since the way we configure ingress apisix is changed.

**Comment by gxthrj** [comment]
> @gxthrj After this PR is merged, the dev docker image should be updated, also, docs and samples should be changed, since the way we configure ingress apisix is changed.

Should we update the docs in this PR?

File: pkg/kube/init.go (hunk 3 above)
Comment (tokers) [reply pkg/kube/init.go:61]: I have run it in my local minikube, it works well.

**Comment by tokers** [comment]
> > @gxthrj After this PR is merged, the dev docker image should be updated, also, docs and samples should be changed, since the way we configure ingress apisix is changed.
> 
> Should we update the docs in this PR?

That's OK.

**Comment by tokers** [comment]
> > > @gxthrj After this PR is merged, the dev docker image should be updated, also, docs and samples should be changed, since the way we configure ingress apisix is changed.
> > 
> > 
> > Should we update the docs in this PR?
> 
> That's OK.

@gxthrj Added.

**Pull request merged**